src/search/lp/lp_internals.h

```diff
@@ -21,6 +21,8 @@ enum class LPSolverType;
 
 std::unique_ptr<OsiSolverInterface> create_lp_solver(LPSolverType solver_type);
 
+void set_mip_gap(OsiSolverInterface *lp_solver, double gap);
```

Comment:
Add a comment what this does? I know it sets the MIP gap, but it would probably be a good idea to have a little explanation what this does or why it is necessary somewhere. It could be a short discussion that links to a longer discussion on the tracker.

Reply:
Looks like `extern` is missing.